### PR TITLE
fix: Shadow colors are no longer solid black

### PR DESCRIPTION
**Pull request by MartinZikmund**
﻿GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Application/v1/ColorPalette.xaml [hunk 1]
```diff
@@ -50,6 +50,8 @@
 			<SolidColorBrush x:Key="MaterialBottomNavUncheckedForegroundBrush"
 							 Color="{ThemeResource MaterialOnPrimaryColor}"
 							 Opacity="0.60" />
+			
+			<Color x:Key="MaterialShadowColor">#33000000</Color>
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Application/v1/ColorPalette.xaml:54]: We'll either need to add this as a "ShadowColor" in the [SharedColorPalette](https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Application/v2/SharedColorPalette.xaml) or create lightweight styling keys that can be used in the material styles for each of the v2 control styles that need it

**Comment by kazo0** [comment]
@Mergify backport release/stable/4.0

File: src/library/Uno.Material/Styles/Application/v1/ColorPalette.xaml (hunk 1 above)
Comment (MartinZikmund) [reply src/library/Uno.Material/Styles/Application/v1/ColorPalette.xaml]: Put it as "ShadowColor" in SharedColorPalette, is that sufficient or something else needs to be done please 😅 ?

File: src/library/Uno.Material/Styles/Controls/v1/DatePicker.xaml [hunk 2]
```diff
@@ -108,7 +108,7 @@
 										  BorderThickness="{TemplateBinding BorderThickness}"
 										  CornerRadius="{TemplateBinding CornerRadius}"
 										  Elevation="{StaticResource MaterialDatePickerFlyoutElevation}"
-										  ShadowColor="Black">
+										  ShadowColor="{StaticResource MaterialShadowColor}">
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v1/DatePicker.xaml:111]: Does MaterialShadowColor exist in the v1 material color palette

Comment (kazo0) [reply src/library/Uno.Material/Styles/Controls/v1/DatePicker.xaml:111]: Basically we should have a `MaterialShadowColor` in the v1 palette and all v1 styles should reference that one. The v2 styles should reference a `ShadowColor` in the SharedColorPalette for v2

**Comment by MartinZikmund** [comment]
@kazo0 Should be all good now!

**Review by kazo0 — APPROVED** [review]

**Pull request merged**